Flutter_TDD_Architecture_Course_notes.md

```diff
@@ -189,6 +189,48 @@ The code will make the first `test` pass so there will be no more errors.
 
 ![First_Test_Pass](https://user-images.githubusercontent.com/27420533/74528160-6d9fef80-4f1f-11ea-9198-64d1aaefb3c4.png)
 
+Full code of `get_concrete_number_trivia_test.dart` file.
```

Comment:
Miguel, when I wrote that you needed to include the "full code" it meant the code needs to be committed to GitHub and _linked_ to from the Markdown file, it does not mean copy pasting the code (duplicating) into the markdown file. Thankfully you have overcome your obstacle to committing and pushing code to GitHub so you will no longer _need_ to paste the code into markdown files and can instead follow the examples provided where the _link_ to the code snapshot is included at the end of each section.